ntbs-service/Pages/Notifications/NotificationsBase.cshtml.cs

```diff
@@ -110,18 +96,18 @@ private IActionResult RedirectToOverview()
             return RedirectToPage("../Overview", new {id = NotificationId});
         }
 
-        protected void SetNotificationProperties<T>(bool isBeingSubmitted, T ownedModel) where T : ModelBase 
+        protected async Task SetNotificationProperties<T>(bool isBeingSubmitted, T ownedModel) where T : ModelBase
         {
             NotificationId = Notification.NotificationId;
             Notification.SetFullValidation(Notification.NotificationStatus, isBeingSubmitted);
             ownedModel.ShouldValidateFull = Notification.ShouldValidateFull;
+
+            await GetLinkedNotifications();
         }
 
-        public ContentResult ValidateModelProperty<T>(string key, object value, bool shouldValidateFull) where T : ModelBase
+        public bool IsValid(string key)
```

Comment:
Needed to add this method as this was called directly from validationService before